app/build.gradle

```diff
@@ -1,8 +1,8 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 29
-    buildToolsVersion "29.0.3"
+    compileSdkVersion 30
```

Comment:
These changes are not needed to solve the addressed issue. You can raise separate pull requests for different changes that you want to propose

Reply:
True, I did that because I think sdk version 29 isn't installed on my setup, so it popped up with build errors. I can change it back if required.

Reply:
Yeah, please remove these changes. Everything else is fine

Reply:
Will do.